src/OpenTelemetry/Metrics/MetricPoint.cs

```diff
@@ -21,37 +21,45 @@
 
 namespace OpenTelemetry.Metrics
 {
+    /// <summary>
+    /// Stores details about a metric data point.
+    /// </summary>
     public struct MetricPoint
     {
         private readonly AggregationType aggType;
+
         private readonly HistogramBuckets histogramBuckets;
 
         // Represents either "value" for double/long metric types or "count" when histogram
-        private MetricPointValueStorage primaryValue;
+        private MetricPointValueStorage runningValue;
+
+        // Represents either "value" for double/long metric types or "count" when histogram
```

Comment:
```suggestion
        // Represents temporality adjusted "value" for double/long metric types or "count" when histogram
```

Reply:
Updated. Just to confirm, you intended for it to be "temporality " in the comment and not "temporarily" right?

Reply:
Yes "Temporality" (Delta vs Cumulative).